README.md

```diff
@@ -123,6 +123,21 @@ The above says that the page titled "US-RSE Community Building Workshop" served
 at permalink /2020-april-workshop will be redirected to 
 https://us-rse.org/first-community-workshop.
 
+### 5. How do I embed a video?
+
+If you have a YouTube video to embed, we have an include that will make it easy
+to embed a full width, fullscreen enabled video! Simply do:
+
+```
+{% include youtube-embed.html url="https://www.youtube.com/embed/gP5UCfV3n-A"  title="Video Title" %}
```

Comment:
I think it's easier for the user to copy the entire link and not have to think about what is the id, but thanks for the suggestion.

Reply:
@coolbrg I disagree that it's "recommended" to use dummy values in the example - with a string like the one in your example the user doesn't have any sense of what the code should look like, because it looks nothing like it. I'm going to resolve this comment because I don't think this suggestion would improve the PR, but thank you for suggesting it!